server/test/list_objects.go

```diff
@@ -51,24 +53,151 @@ func newListObjectsRequest(store, objectType, relation, user, modelID string, co
 	}
 }
 
-func TestListObjects(t *testing.T, datastore storage.OpenFGADatastore) {
-	store := testutils.CreateRandomString(10)
+func ListObjectsTest(t *testing.T, ds storage.OpenFGADatastore) {
+
+	ctx := context.Background()
 	tracer := telemetry.NewNoopTracer()
-	ctx, backend, modelID, err := setupTestListObjects(store, datastore)
-	require.NoError(t, err)
 
-	t.Run("list objects", func(t *testing.T) {
+	t.Run("Github without TypeInfo", func(t *testing.T) {
+		store := id.Must(id.New()).String()
+
+		data, err := os.ReadFile(gitHubTestDataFile)
+		require.NoError(t, err)
+
+		var gitHubTypeDefinitions openfgapb.WriteAuthorizationModelRequest
+		err = protojson.Unmarshal(data, &gitHubTypeDefinitions)
+		require.NoError(t, err)
+
+		model := &openfgapb.AuthorizationModel{
+			Id:              id.Must(id.New()).String(),
+			SchemaVersion:   typesystem.SchemaVersion1_0,
+			TypeDefinitions: gitHubTypeDefinitions.GetTypeDefinitions(),
+		}
+		err = ds.WriteAuthorizationModel(ctx, store, model)
+		require.NoError(t, err)
+
+		writes := []*openfgapb.TupleKey{tKAllAdminsRepo6, tkAnnaRepo1, tkAnnaRepo2, tkAnnaRepo3, tkAnnaRepo4, tkBobRepo2}
+		err = ds.Write(ctx, store, nil, writes)
+		require.NoError(t, err)
+
+		testCases := []listObjectsTestCase{
```

Comment:
This is duplicate in the other test

Reply:
`model.Id` differs because the others have to be against v1.1 models. I'll think through how I can refactor to reduce duplication.